app/controllers/videos_controller.rb

```diff
@@ -63,7 +60,10 @@ def destroy
   private
     # Use callbacks to share common setup or constraints between actions.
     def set_video
-      @video = Video.find(params[:id])
+      @video = Video.find_by(:id)
```

Comment:
@moizansari22 Did you test it?

Reply:
again implemented and tested and pushed the code to a new commit named as "Latest incorporated feedback"